pkg/sfu/streamtracker.go

```diff
@@ -106,9 +143,10 @@ func (s *StreamTracker) Observe(sn uint16) {
 	}
 
 	if !s.initialized.get() {
-		s.initialized.set(true)
+		s.lastSN = sn
 		// first packet
-		go s.setStatus(StreamStatusActive)
+		go s.init()
```

Comment:
this would mean the next Observe could still call s.init() as go spins up another goroutine. I think it'll also contain a race as `s.running = make(chan struct{})` would be set in a thread without lock.

I didn't fully understand why we'd want to. do this implicitly vs explicitly with Start.

Reply:
Will have another look at it David. I wanted the timer based detect to kick in after unit. That's why I did it on first packet.

Reply:
ah I see, in that case we can use a mutex